go/validation/tests/databricks.py

```diff
@@ -0,0 +1,91 @@
+# Copyright (c) 2025 ADBC Drivers Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pathlib import Path
+
+from adbc_drivers_validation import model, quirks
+
+
+class DatabricksQuirks(model.DriverQuirks):
+    name = "databricks"
+    driver = "adbc_driver_databricks"
+    driver_name = "ADBC Driver Foundry Driver for Databricks"
+    vendor_name = "Databricks"
+    vendor_version = "v1.9.0"
+    short_version = "v1.9.0"
```

Comment:
It seems this is the driver version and not the vendor version; that needs to be fixed